.github/workflows/main.yml

```diff
@@ -246,17 +246,10 @@ jobs:
         uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4.0.1
         with:
           node-version: ${{ env.NODE_VERSION }}
-      - name: Set up Emscripten
-        uses: mymindstorm/setup-emsdk@6ab9eb1bda2574c4ddb79809fc9247783eaf9021 # v14
-        with:
-          version: ${{ env.EMSCRIPTEN_VERSION }}
```

Comment:
Removing the Emscripten setup shaves off ~35 seconds from the CI job timings, and will reduce it more if the caches are evicted/invalidated. It isn't a lot, but I hope it's a welcome change. :)